moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -731,6 +731,10 @@ bool PlanningSceneMonitor::newPlanningSceneMessage(const moveit_msgs::msg::Plann
 
     if (!scene.is_diff && parent_scene_)
     {
+      // if the scene does not contain a robot state then use the latest value from the current_state_monitor_
+      // so that we are not updating the scene with stale values for the robots state.
+      if (scene.robot_state.joint_state.name.empty())
+        parent_scene_->setCurrentState(*current_state_monitor_->getCurrentState());
```

Comment:
Can you put braces around this? The single-line if-statement body has bitten me many a time.